prometheus_xmpp/__main__.py

```diff
@@ -179,14 +232,125 @@ def message(self, msg):
                 response = "Unknown command: %s" % args[0].lower()
             msg.reply(response).send()
 
+    async def dispatch_message(self, mto, mbody, mhtml):
```

Comment:
it seems this function ignores mto completely if self.muc is set, which is confusing; cam we just pass in self.muc_jid ?